package.json

```diff
@@ -69,7 +69,7 @@
   "scripts": {
     "knit": "knit",
     "build": "knit build --skip-preflight --verbose",
-    "start": "knit server",
+    "start": "rm data/schema.json&&rm data/version.json&&knit schema&&knit server",
```

Comment:
This will fail if the files don't already exist because rm will fail. probably should replace `&&` with `;`